### PR TITLE
Add npmRegistry configuration option

### PR DESCRIPTION
**Pull request by developit**
See https://github.com/zeit/hyperterm/pull/211

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -541,6 +541,11 @@ <h2 id="cfg"><a href="#cfg">Configuration</a></h2>
               <td>""</td>
               <td>A path to a custom shell to run when hyperterm starts a new session</td>
             </tr>
+            <tr>
+              <td>"npmRegistry"</td>
+              <td>npm get registry</td>
+              <td>Override the NPM registry URL to use when installing plugins</td>
```
Comment (sindresorhus) [inline index.html]: `NPM` => `npm`

Comment (developit) [inline index.html]: Ahh how did I let thst slip

**Comment by rauchg** [comment]
Waiting for `NPM` fix to merge <?>

**Comment by developit** [comment]
Ah sorry, updated!

**Pull request merged**